io.github.adenlall.salatok-gtk.yml

```diff
@@ -0,0 +1,28 @@
+app-id: io.github.adenlall.salatok-gtk
+runtime: org.gnome.Platform
+runtime-version: '44'
```

Comment:
Can't it use the latest version of the runtime? I.e. version `45`?

Reply:
Thank you I have fixed these lines and add SDK runtime to the latest 45